CHANGELOG/v1.7.5.md

```diff
@@ -0,0 +1,51 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.4
+## :chart_with_upwards_trend: Overview
+- 20 new commits merged
+- 1 feature addition ✨
+- 6 bugs fixed 🐛
+
+## :sparkles: New Features
+- ClusterCacheTracker: Add QPS & burst options & flags for ClusterCacheTracker (#10894)
+
+## :bug: Bug Fixes
```

Comment:
Pleas also include https://github.com/kubernetes-sigs/cluster-api/pull/11043/files here

It will merge in a few minutes

Reply:
Done.